liquigraph-core/src/test/java/org/liquigraph/core/configuration/ConfigurationBuilderTest.java

```diff
@@ -57,7 +58,7 @@ public void fails_on_non_existing_changelog() {
     @Test
     public void fails_on_null_uri() {
         thrown.expect(RuntimeException.class);
-        thrown.expectMessage("- Exactly one of JDBC URI or DataSource need to be configured");
+        thrown.expectMessage("- " + INCORRECT_CONFIGURATION_OPTION_ERROR_MESSAGE);
```

Comment:
optional: duplicate the message here, it makes the test code more straightforward to read (although it'll fail whenever the error change, but that's kind of an API change so it's OK imo)